Sec borg and peacemaker improvement

Sinmero opened the pull request:
<!-- Write **BELOW** The Headers and **ABOVE** The comments else it may not be viewable. -->
<!-- You can view Contributing.MD for a detailed description of the pull request process. -->

## About The Pull Request

Secborg has a very poor amount of tools in their disposal compaired to other borgs. This pr fixes it.
Adds security holobarrier projector, forensic scanner, pepper spray and evidence bag to secborg.
Items will drop out of evidence bag if borg module is reset or borg is destroyed or if it was emped.

Peacekeeper harmalarm is in a bad place atm. It confuses people causing them to wallslam which is human harm and against the whole point of peaceborg.
Peacekeeper harmalarm will no longer confuse but knockdown people for 3 seconds.

<!-- Describe The Pull Request. Please be sure every change is documented or this can delay review and even discourage maintainers from merging your PR! -->

## Why It's Good For The Game

Secborg has a wider array of tools and can investigate and collect evidence now.
Peacemaker's harmalarm is a bit better at preventing harm and doesnt cause human harm now.

<!-- Please add a short description of why you think these changes would benefit the game. If you can't justify it in words, it might not be worth adding. -->

## Changelog
:cl:
add: Added security holobarrier projector, forensic scanner, pepper spray and evidence bag to secborg.
tweak: Peacekeeper harmalarm will knockdown people for 3 seconds instead of confusing them.
/:cl:

<!-- Both :cl:'s are required for the changelog to work! You can put your name to the right of the first :cl: if you want to overwrite your GitHub username as author ingame. -->
<!-- You can use multiple of the same prefix (they're only used for the icon ingame) and delete the unneeded ones. Despite some of the tags, changelogs should generally represent how a player might be affected by the changes rather than a summary of the PR's contents. -->

Sinmero committed:
Update

Fronsis commented:
I guess Secborg being able to investigate can be.. an interesting gimmick to pick and help security instead of going BAM BAM UNLIMITED DISABLE LASER WOO specially when there's no detective or.. well the detective doesn't do his job and just likes to use revolver..

As for Peacekeeper hmm that's VERY debatable.. from what i've seen most Peacekeeper borgs lately just... don't feel useful, obviously back on the old HUMAN HARM alarm you could make everybody wall-slam and trip and that was kinda overpowered but ever since the wall-slam nerf and changes to stamina and stuns.. Most peacekeepers i see just... give cookies, meme around and try to use the HUMAN HARM alarm but.. they don't do anything

I think i'd like to have a 1 second stun that makes people drop items, gives chance for the Borg to actually help when it witness human/crew harm and could possibly save someone being murdered and since Sec use headset that would come in handy, of course it won't reach levels of disgusting stun like HONKBLASTER but.. it's a good start, both borgs have been at the bottom of usefulness from what i've seen and experienced these last months

ghost commented:
>evidence bag

Unless you add a way to strip items from borgs, this will introduce a storage that nobody but the borg can access. Perfect for hiding nuke disk.

ItsTisme commented:
Maby a module for the borg to scan, it would take aslong as regulair searching but would help because borgs can only strip items not putt hem back on

eeSPee commented:
>holobarrier projector

Sure.

> forensic scanner

Kinda conflicted. On one side, it gives the secborg more to do, on the other side, it makes detective useless.

>evidence bag

No. For the reasons stated above.

Naevii commented:
"Also peacekeeper harmalarm will very briefly stun people and make them drop items in hands."
I don't see this ability being used in any good and usefull way. 
peacekeepers already use harm alarm when its not necessary and there is no harm (because borgs tend to over-inpretet or just do it for funies) 

this would only make it abusable.

ghost commented:
>it makes detective useless.

Security lathe can print infinite forensic scanners, most likely sooner than security borgs even appear into a round.

Fronsis commented:
> "Also peacekeeper harmalarm will very briefly stun people and make them drop items in hands."
> I don't see this ability being used in any good and usefull way.
> peacekeepers already use harm alarm when its not necessary and there is no harm (because borgs tend to over-inpretet or just do it for funies)
> 
> this would only make it abusable.

Agree, lately i've been seen a LOT of Peacekeeper using it for no reason but.. that should be clarified and more enforced as a rule.

utilbird commented:
Orrr maybe giving borgs an AOE hardstun/item drop is a bad idea

ghost commented:
people are complaining that secborg is OP and they want to buff them further?
giving a forensic scanner to he secborg would make them infinitly better than an actual detective
>AA
>Max speed
>Forensic scanner
recipe for disaster
also that aoe harm alarm thing is a funny meme

Sinmero committed:
fix

Sinmero commented:
> Unless you add a way to strip items from borgs, this will introduce a storage that nobody but the borg can access. Perfect for hiding nuke disk.

Items will drop out of evidence bag if borg module is reset or borg is destroyed now.

>I don't see this ability being used in any good and usefull way.
peacekeepers already use harm alarm when its not necessary and there is no harm (because borgs tend to over-inpretet or just do it for funies)
this would only make it abusable.

>Orrr maybe giving borgs an AOE hardstun/item drop is a bad idea

Removed stun. Confusion causes human harm because of wallslam. Replaced confusion with knockdown.

>people are complaining that secborg is OP and they want to buff them further?
giving a forensic scanner to he secborg would make them infinitly better than an actual detective

I personally dont believe that secborgs are OP. In any case this pr doesnt buff their 'detaining' ability but gives them more things to do.

ghost commented:
> Items will drop out of evidence bag if borg module is reset or borg is destroyed now.

That's not sufficient in my opinion. Imagine you had to kill or perform surgery on someone just to take an item from their hands.

If you want a borg to have a hand to hold any item, or just hold them internally, there has to be a way to strip them without such invasive methods. It will be abused in combat and people will order borgs to permanently hide high risk items.

I would suggest instead being able to deploy an external item carrier, such as a bodybag to put items or bodies in and drag behind.

Sinmero commented:
>That's not sufficient in my opinion. Imagine you had to kill or perform surgery on someone just to take an item from their hands.
If you want a borg to have a hand to hold any item, or just hold them internally, there has to be a way to strip them without such invasive methods. It will be abused in combat and people will order borgs to permanently hide high risk items.

Evidence bag can store only one item. Scince its a sec borg i think its appropriate that the storage would be somewhat secure.
We do have secure briefcases and secure safes that prevent people from taking items withing them.

You can always law 2 order borg to drop the item they are holding.
I dont think rules would allow to take high risk items. And again they can be ordered to drop it.
Borgs wont be effiecent at protecting the disk either (compaired to an armed human). Nukie can just emp them or flashbang, flash, emag or just change laws.

And we do have grip implant that would require surgery to remove the item from hand.

ghost commented:
>secure briefcases and secure safes that prevent people from taking items withing them

They can be hacked with regular tools from a lathe.

Sinmero commented:
>They can be hacked with regular tools from a lathe.

Borg can be reset with regular tools from lathe. Crowbar screwdriver and wirecutters is all it takes.

ghost commented:
> Borg can be reset with regular tools from lathe. Crowbar screwdriver and wirecutters is all it takes.

I hope you're arguing in good faith, because this is false. You need to force the borg to unlock the cover, have correct access for it, or kill the borg.

Sinmero committed:
emp

Raven-Industries commented:
Please do not give secborg a forensic scanner. Detectives already have to struggle against indifferent security and janitors cleaning up the crime scenes near-instantly, we don't need the robots taking their job too.

ghost commented:
> Please do not give secborg a forensic scanner. Detectives already have to struggle against indifferent security and janitors cleaning up the crime scenes near-instantly, we don't need the robots taking their job too.

Forensic scanner is not a unique item to detective, anyone with access to security lathe can print them.

Raven-Industries commented:
Luckily it seems very few are aware of that.

bloons3 commented:
> Please do not give secborg a forensic scanner. Detectives already have to struggle against indifferent security and janitors cleaning up the crime scenes near-instantly, we don't need the robots taking their job too.

Detective also has to struggle against the HoS stealing his forensic scanner to go validhunting anyways

Fronsis commented:
Eitherway... that can always be resolved because... lathe

Cenrus commented:
Can you just rework secborg into a detective borg instead?

Sinmero commented:
> Can you just rework secborg into a detective borg instead?

I already gave them most of det tools.

utilbird commented:
Peacekeepers are a meme module, they're not supposed to be strong or validhunt-y.
Sec borgs are already stupidly strong and versatile, they certainly don't need anything new. Detective is ignored enough as is anyway 😓
